lib/BootBot.js

```diff
@@ -425,35 +425,39 @@ class BootBot extends EventEmitter {
     data.entry.forEach((entry) => {
       // Iterate over each messaging event
       entry.messaging.forEach((event) => {
-        if (event.message && event.message.is_echo && !this.broadcastEchoes) {
-          return;
-        }
-        if (event.optin) {
-          this._handleEvent('authentication', event);
-        } else if (event.message && event.message.text) {
-          this._handleMessageEvent(event);
-          if (event.message.quick_reply) {
-            this._handleQuickReplyEvent(event);
-          }
-        } else if (event.message && event.message.attachments) {
-          this._handleAttachmentEvent(event);
-        } else if (event.postback) {
-          this._handlePostbackEvent(event);
-        } else if (event.delivery) {
-          this._handleEvent('delivery', event);
-        } else if (event.read) {
-          this._handleEvent('read', event);
-        } else if (event.account_linking) {
-          this._handleEvent('account_linking', event);
-        } else if (event.referral) {
-          this._handleEvent('referral', event);
-        } else {
-          console.log('Webhook received unknown event: ', event);
-        }
+        this.handleFacebookMessage(event);
```

Comment:
`handleFacebookEntry()`? Fits facebooks API better and maybe easier to translate for people who actually know what their data looks like (the assumed clients of this method).